src/ui/theme.h

```diff
@@ -165,6 +170,7 @@ namespace ui {
                            gfx::Size& sizeHint,
                            gfx::Border& borderHint,
                            gfx::Rect& textHint, int& textAlign);
+    static ui::Style* m_defaultStyle;
```

Comment:
Use a unique pointer so it's deleted automatically:
```suggestion
    static std::unique_ptr<ui::Style> m_defaultStyle;
```

Reply:
Done.